src/main/java/pl/touk/sputnik/processor/ReviewProcessorFactory.java

```diff
@@ -0,0 +1,11 @@
+package pl.touk.sputnik.processor;
+
+import pl.touk.sputnik.configuration.Configuration;
+import pl.touk.sputnik.review.ReviewProcessor;
+
+public interface ReviewProcessorFactory<T extends ReviewProcessor> {
+
+    boolean isEnabled(Configuration aConfiguration);
```

Comment:
Just use configuration instead of aConfiguration. Here and in implemented classes.